docs/v1/simple-service-broker-writing.md

```diff
@@ -0,0 +1,31 @@
+# Use Cases For Simple Service Broker Writing
+Jay Judkowitz
+
+Nov 7th, 2016
+
+## Introduction
+At the Catalog SIG offsite, I brought up the topic that we need to have some way to make service broker authoring simple
+(whether part of the Catalog deliverables or separate) or that we may not get people to adopt the technology.  There was some 
+pushback that this was already handled and not a challenge and that I should put forth use cases that we can test the current 
+plans against.  This document is the response to that.
+
+## Multi-tenancy
+It is clear that we can have some default service broker that always returns the same credentials for all binding apps.  This 
+makes perfect sense for single tenant applications.  The credentials can be determined by the service instantiator at 
+provisioning time and those can be stashed away and returned on any bind call.  But, what do we do for multi-tenant services?  
+
+What is the easiest way to write a broker that can create unique credentials for each bind call
+* Possibly creating everything itself
+* Possibly some mix of creating things itself and taking inputs from the binding application.  For example, imagine the bind 
+call schema asking the binding application to pass in a new username, but then the service broker generates the password and 
+both the username and password are stored as secrets.
+
+## Implementing binding mechanics
+How does the service provider implement binding specific logic that goes beyond simply passing back credentials?  Perhaps the 
+service needs to provision some capacity, create database entries, create new users, etc….   We need a way to store that per 
+binding application logic.
```

Comment:
I do not totally agree that this is entirely out of scope for the sig.  I think making a skeletal broker is something that will probably eventually happen naturally.  We also will need a mock-broker that implements the SB API to test against.  IMO the bits that respond to API requests might be generic enough to publish as a library.  With that said, I do think that providing broker libraries in multiple language is not in scope, and making a complex generic broker that has its own inner platform to be everything to everyone is not a goal.